tests/conftest.py

```diff
@@ -73,7 +73,7 @@ def meilisearch_container(meilisearch_port):
             continue
 
     # yield the container
-    yield container
+    return container
```

Comment:
```suggestion
    yield container
```

This is the only way to ensure that things run properly once the session finishes @Revathyvenugopal162